src/routes/account/+page.server.ts

```diff
@@ -53,8 +55,7 @@ export const load = (async ({ url, cookies }) => {
         throw redirect(303, stripeSession.url!);
       }
       case 'choosePlan':
-        url.searchParams.delete('action');
-        throw redirect(303, `/account/plan?${url.searchParams.toString()}`);
```

Comment:
This was throwing an error in the new Sveltekit version. I think maybe they intentionally made `url.searchParams` read only on the server so that you don't edit them thinking it will edit the browser search params. Instead I now create new params above.